### PR TITLE
Match assert number message format with code format

### PR DESCRIPTION
**Pull request by anthony-murphy**
related to #5498

### CODE REVIEW

**Comment by anthony-murphy** [comment]
@jatgarg i'd like this to make the 0.37 release

**Comment by anthony-murphy** [comment]
ping @markfields

File: common/lib/common-utils/src/assert.ts [hunk 1]
```diff
@@ -12,6 +12,6 @@
  */
  export function assert(condition: boolean, message: string | number): asserts condition {
      if (!condition) {
-         throw new Error(typeof message === "number" ? message.toString(16) : message);
+         throw new Error(typeof message === "number" ? `0x${message.toString(16).padStart(3,"0")}` : message);
```
Comment (markfields) [inline common/lib/common-utils/src/assert.ts:15]: unit test to prove it?

**Review by markfields — APPROVED** [review]
🎂

**Pull request merged**